test/unit/lib/adapters/tool/testplane/index.ts

```diff
@@ -340,6 +340,113 @@ describe('lib/adapters/tool/testplane/index', () => {
         });
     });
 
+    describe('runWithoutRetries', () => {
```

Comment:
Use tests from `test/lib/gui/app`. Here I just brought them to the necessary form